TthCrstl/README.txt

```diff
@@ -1,9 +0,0 @@
-Tythonian Crystal

```

Comment:
If you're going to rename files, you need to use "git mv" or something so that the diff works properly.
This way it's really hard to see what you changed/added from the original file.